recipes/sasl/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{%set version = "0.2.1"%}
+
+package:
+  name: sasl
+  version: {{ version }}
+
+source:
+  fn: sasl-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/sasl/sasl-{{ version }}.tar.gz
+  md5: ca093d9a3d6f20b79b964a5e5add0202
+
+build:
+  number: 0
+  script: python setup.py install  # [osx]
```

Comment:
Given the lack of need for the `apt-get` command, you can remove the pre-processor from here and remove the `build.sh` and `bld.bat` files in favor of this script. Also, this is missing `--single-version-externally-managed --record=record.txt`